dnf-behave-tests/dnf/dnf-automatic/config.feature

```diff
@@ -0,0 +1,20 @@
+@no_installroot
+Feature: dnf-automatic configuration files testing
+
+Background:
+Given I delete file "/etc/yum.repos.d/*.repo" with globs
+  And I create file "/etc/dnf/dnf.conf" with
+    """
+    [main]
+    plugins=0
+    """
```

Comment:
I think this is not needed here but I am fine with keeping it.

Could you add a comment or name the background (`Backgroud: setup clean environment`) to make its purpose clear?

Reply:
Right. It's a copy paste from another automatic test, but it's not needed here. I'll remove the background section. It doesn't even need to be run outside of the installroot.